core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/dr/BackupRestoreSaltConfigGenerator.java

```diff
@@ -43,7 +52,12 @@ public SaltConfig createSaltConfig(String location, String backupId, String rang
         disasterRecoveryValues.put(OBJECT_STORAGE_URL_KEY, fullLocation);
         disasterRecoveryValues.put(RANGER_ADMIN_GROUP_KEY, rangerAdminGroup);
         disasterRecoveryValues.put(CLOSE_CONNECTIONS, String.valueOf(closeConnections));
-
+        if (!skipDatabaseNames.isEmpty()) {
+            ArrayList<String> names = new ArrayList<>();
+            DEFAULT_BACKUP_DATABASE.forEach(db -> names.add(db.toString().toLowerCase()));
+            names.removeAll(skipDatabaseNames);
```

Comment:
Do we want validation anywhere to ensure the names here are valid? I know they just won't be removed but just wondering if it's worth to at least log.

Reply:
Logged